Apps/WaitingQueueWeb/Controllers/TicketController.cs

```diff
@@ -47,19 +45,16 @@ public TicketController(IWebTicketDelegate ticketDelegate)
         /// <returns>A ticket response when successful.</returns>
         /// <param name="room">The room for which the client is requesting a ticket.</param>
         /// <response code="200">Ticket returned.</response>
-        /// <response code="400">The request was invalid.</response>
         /// <response code="404">The requested room was not found.</response>
-        /// <response code="429">The user has made too many requests in the given timeframe.</response>
         /// <response code="503">The service is too busy, retry after the amount of time specified in retry-after.</response>
         [HttpPost]
         [ProducesResponseType(typeof(Ticket), StatusCodes.Status200OK)]
-        [ProducesResponseType(StatusCodes.Status400BadRequest)]
-        [ProducesResponseType(typeof(ErrorResult), StatusCodes.Status404NotFound)]
-        [ProducesResponseType(StatusCodes.Status429TooManyRequests)]
-        [ProducesResponseType(typeof(ErrorResult), StatusCodes.Status503ServiceUnavailable)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status503ServiceUnavailable)]
+
         public async Task<IActionResult> CreateTicket([FromQuery]string room)
```

Comment:
I thought we were going to remove IActionResult and simply replace with the Ticket type?

Reply:
That was only for the delegate.  In the controller it has to be IActionResult because the controller can return different ActionTypes example, OK result object with Ticket or a 404 result or a 503.  See [doc](https://learn.microsoft.com/en-us/aspnet/core/web-api/action-return-types?view=aspnetcore-6.0#iactionresult-type-1)